Bottom: 7f1789a6f300f6169804aca937fdcf00243d71a5
Top:    ede6d83c75e1553610628d14f2d91e4ed23f7274
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 14:02:54 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/env.yml b/roles/buildbot-master/env.yml
deleted file mode 100644
index 7ef0578f..00000000
--- a/roles/buildbot-master/env.yml
+++ /dev/null
@@ -1,22 +0,0 @@
-BUILDBOT_VERSION=1.1.1
-BUILDBOT_UID=1000
-
-BUILDBOT_VOL_CACHE=/home/coopera/Developer/buildbot-mounts/cache:/cache
-BUILDBOT_VOL_MASTER=/home/coopera/Developer/buildbot-mounts/buildbot:/buildbot
-BUILDBOT_VOL_NTEL=/home/coopera/Developer/buildbot-mounts/worker-ntel:/buildbot
-BUILDBOT_VOL_PTXDIST=/home/coopera/Developer/buildbot-mounts/worker-ptxdist:/buildbot
-BUILDBOT_VOL_SECRETS=./secrets:/run/secrets:ro
-
-BUILDBOT_DATABASE=sqlite:////cache/state.sqlite
-BUILDBOT_MASTER=buildbot
-BUILDBOT_TIMEZONE=America/Chicago
-BUILDBOT_HOST_PORT=172.16.103.71:80
-BUILDBOT_WEB_PORT=8010
-BUILDBOT_WEB_URL=http://buildbot.novatech-llc.com/
-
-BUILDBOT_WORKER_NTEL=worker-ntel
-BUILDBOT_WORKER_PASSWORD=pass
-BUILDBOT_WORKER_PORT=9989
-BUILDBOT_WORKER_PTXDIST=worker-ptxdist
-
-BUILDBOT_ENVIRONMENT_BLACKLIST=DOCKER_BUILDBOT*,BUILDBOT_ENV_*,BUILDBOT_1*,BUILDBOT_ENVIRONMENT_BLACKLIST,BUILDBOT_WORKER_PASSWORD
